### PR TITLE
FIX: InputFile CloseButton triggers file selection

### LINKED ISSUE #459: InputFile_Filled with file - CloseButton triggers file selection in IE
## Expected Behavior

CloseButton should only remove an uploaded file, no other behavior is expected

## Current Behavior

CloseButton triggers a file selection option.
select
## Steps to reproduce
Click the close button 

## Storybook link

https://kiwicom.github.io/orbit-components/?knob-Label=Label&knob-fileName=file.png&selectedKind=InputFile&selectedStory=Filled%20with%20file&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

### PR DESCRIPTION
**Pull request by wEprous**
Fixes #459 <br/><br/><br/><url>LiveURL: https://orbit-components-fix-inputfile-closebutton-trigger.surge.sh</url>

### CODE REVIEW

File: src/InputFile/index.js [hunk 1]
```diff
@@ -130,7 +130,10 @@ const InputFile = (props: Props) => {
             type="secondary"
             transparent
             icon={<CloseCircle />}
-            onClick={props.onRemoveFile}
+            onClick={ev => {
+              ev.preventDefault();
+              return onRemoveFile ? onRemoveFile(ev) : null;
```
Comment (tomashapl) [inline src/InputFile/index.js]:
```
if(onRemoveFile) {
    onRemoveFile();
}
```

no return please

File: src/InputFile/index.js.flow [hunk 2]
```diff
@@ -13,7 +13,7 @@ export type Props = {|
   onChange?: (ev: SyntheticInputEvent<HTMLInputElement>) => void | Promise<any>,
   onFocus?: (ev: SyntheticInputEvent<HTMLInputElement>) => void | Promise<any>,
   onBlur?: (ev: SyntheticInputEvent<HTMLInputElement>) => void | Promise<any>,
-  onRemoveFile?: () => void,
+  onRemoveFile?: (ev: SyntheticEvent<HTMLButtonElement>) => void,
```
Comment (tomashapl) [inline src/InputFile/index.js.flow]:
```
onRemoveFile?: () => void | Promise<any>
```

I can't imagine situation when `ev` of button click is needed

Comment (wEprous) [reply src/InputFile/index.js.flow]: You're right, but we need to flow type the `ev` argument 😞

Comment (tomashapl) [reply src/InputFile/index.js.flow]: if you don't pass in onRemoveFile fci argument `ev` you don't need it `onRemoveFile()`

**Pull request merged**